apps/src/templates/levelSummary/SummaryEntryPoint.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import getScriptData from '@cdo/apps/util/getScriptData';
+import Button from '@cdo/apps/templates/Button';
+import {ViewType} from '@cdo/apps/code-studio/viewAsRedux';
+import i18n from '@cdo/locale';
+import styles from './summary-entry-point.module.scss';
+
+const SUMMARY_PATH = '/summary';
+
+const SummaryEntryPoint = ({students}) => {
+  // If viewing the page as Participant, be sure to rewrite the link URL
+  // to view as Instructor, so we don't just get redirected back.
+  const params = document.location.search.replace(
+    `viewAs=${ViewType.Participant}`,
+    `viewAs=${ViewType.Instructor}`
+  );
+  const summaryUrl = document.location.pathname + SUMMARY_PATH + params;
+
+  const data = getScriptData('freeresponse');
```

Comment:
i definitely copied this over from the entry point js when I moved the react code out of there. looking at the rest of the codebase, there are a ton of things in `apps/src/sites/studio` that are calling it, but only one in `apps/src/templates` outside of what I added for these new components. i'm not sure what the reason is, but i think that's enough of a pattern to justify me moving it back out of here and into the entry points

Reply:
oh... as soon as i started implementing this i realized it makes the unit tests a million times easier since i can get rid of all the script-tag-injection stuff 😅 maybe that's why